docs/proposals/config-dump.md

```diff
@@ -0,0 +1,248 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
```

Comment:
Please add an alternatives section describing what you consider the alternatives - some I can imagine are:

* We just add labels to certain configmaps
* We add annotations to config maps